config/config.exs

```diff
@@ -32,3 +32,9 @@ config :phoenix, :format_encoders,
 config :mime, :types, %{
   "application/vnd.api+json" => ["json-api"]
 }
+
+config :guardian, Guardian,
+  issuer: "MyApp",
+  ttl: { 1, :hours },
+  secret_key: %{"k" => "X5uaThfmNGtgdwaYJJjUFA", "kty" => "oct"},
```

Comment:
I know this is a school project, but ideally would we be worried of having this committed?